### DISCUSSION TITLE
System not configured. Please connect diag tool

### DISCUSSION THREAD

**Discussion by zkmsh** [cedricp/ddt4all#1284] [General]
Hello,

I have a megane 3 with tomtom carminat. I have a message saying "System not configured. Please connect diag tool" on my screen. I use an OBDLink SX with DDT4ALL. I bought an extra OBD plug and modified the wiring so it can communicate with the multimedia ECU. When I go to my audio ECU in accoustic parameters, here is what I get:

<img width="1066" alt="Capture d’écran 2024-11-01 160042" src="https://github.com/user-attachments/assets/ac7ccdfc-e4bd-4403-bf5c-b053c8fecaff">

 I read that I need to write acoustic parameters, I tried using pyren bot but I don't really now where to find these parameters, I know these parameters are in the .cmd file but that's it. 

I'm kinda lost, could you guys help me?

thanks in advance.

**Comment by zkmsh** [comment]
I use the macro with Pyren and I can now hear the radio but the quality is terrible and the screen still show "system not configured, please connect diag tool". 

Here is my log:

[ecu_log.txt](https://github.com/user-attachments/files/17607149/ecu_log.txt)
[elm_log.txt](https://github.com/user-attachments/files/17607150/elm_log.txt)

**Comment by zkmsh** [comment]
I tried to ask Pyren bot for macros again (the last time I did, it was in January 2024), and the bot return nothing except empty folders, do you know why? @shrlnm (my VIN is VF1BZ1G0643457042)

**Reply by shrlnm** [reply]
@zkmsh Please try once again

**Reply by zkmsh** [reply]
Hello @shrlnm,

Now it doesn't even respond when I request vin.

**Reply by shrlnm** [reply]
@zkmsh Now it's working. Sometimes the server stops
